crypto/x509/x509_acert.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+#include <crypto/x509_acert.h>
```

Comment:
personally, I will put this #include inside the guards, not outside.

Reply:
Ok. I can do that.